gui/slick/js/ajax-episode-subtitles.js

```diff
@@ -66,8 +66,8 @@
 
 (function() {
     $.ajaxEpRedownloadSubtitle = function() {
-        $('.epRedownloadSubtitle').on('click', function() {
-            event.preventDefault();
+        $('.epRedownloadSubtitle').on('click', function(evt) {
+            evt.preventDefault();
```

Comment:
We name it `e` for consistency, that's to prevent the link actually going to the url.